src/test/com/fulcrologic/fulcro/routing/legacy_ui_routers_spec.cljc

```diff
@@ -18,10 +19,24 @@
    :router-targets {:PAGE/simple-target SimpleTarget}
    :router-id      :PAGE/root-router})
 
+(comp/defsc Root [this _]
+          {:query         [{:root-router (comp/get-query SimpleTarget)}]
+           :initial-state (fn [& _]
```

Comment:
I would prefer not to advertise this as a "correct" way to spec args for initial state.

Reply:
Not Desired. Will remove